lib/rules/no-bitwise.js

```diff
@@ -36,12 +38,21 @@ module.exports = function(context) {
     }
 
     /**
+     * Checks if exceptions were provided, e.g. `{ allow: ['~', '|'] }`.
+     * @param   {ASTNode} node The node to check.
+     * @returns {boolean} Whether or not the node has a bitwise operator.
+     */
+    function allowedOperator(node) {
```

Comment:
How about `isOperatorAllowed` as the name of the function? Then pass in the operator as the argument to his function. Its more readable IMO.